app.js

```diff
@@ -0,0 +1,43 @@
+const bookForm = document.querySelector('#book_form');
+const bookContainer = document.querySelector('.books');
+
+let bookList = [];
+// GET BOOKS FROM LOCAL STORAGE
+const getData = () => {
+  const data = localStorage.getItem('books');
+  if (data) {
+    const parsedData = JSON.parse(data);
+    bookList = parsedData;
+    bookContainer.innerHTML = parsedData.map((book) => `
+                <h3>${book.titleInput}</h3>
+                <p>${book.authorInput}</p>
+                <button type="button"  onclick='deleteBook(${book.id})'>Remove</button>
+            <hr>
+            `).join('');
+  }
+};
+// ADD BOOKS FROM LOCAL STORAGE
+bookForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const titleInput = document.getElementById('title').value;
+  const authorInput = document.getElementById('author').value;
+  const id = Math.floor(Math.random() * 1000);
+  const book = { id, titleInput, authorInput };
+  bookList.push(book);
+  // console.log(bookList);
```

Comment:
[Optional] Let's remove all console logs to make the code cleaner and professional ✊